openpype/hosts/nuke/api/lib.py

```diff
@@ -563,7 +563,7 @@ def get_node_path(path, padding=4):
 
 
 def get_nuke_imageio_settings():
-    return get_anatomy_settings(Context.project_name)["imageio"]["nuke"]
+    return get_project_settings(Context.project_name)["nuke"]["imageio"]
```

Comment:
I believe there should be some backward compatibility check. Some clients might have Anatomy ImageIO set from 3.9 and they didn't change `__legacy__` type yet. We would need to go trough all projects and do this manually (impossible). 
For those projects it doesnt even make sense to copy paste it from Anatomy to `./nuke/imageio`.

I will do the changes in separate PR. Please could you return the `___legacy__` check (https://github.com/pypeclub/OpenPype/pull/3909#discussion_r979023257)